src/components/Insights/HeatMap.js

```diff
@@ -357,15 +348,14 @@ export const HeatMap = React.createClass({
     this.clearMap();
     this.status = "loading";
     const siteKey = this.props.siteKey;
-    this.moveMapToNewLocation(state.selectedLocationCoordinates, this.map.getZoom());
     const zoom = this.map.getZoom();
     const bbox = this.getLeafletBbox();
     let self = this;
     this.weightedMeanValues = [];
 
     if(state.categoryValue.name){
-        SERVICES.getHeatmapTiles(siteKey, state.timespanType, zoom, state.categoryValue.name, state.datetimeSelection, 
-                                bbox, Array.from(state.termFilters), [state.selectedLocationCoordinates], Actions.DataSources(state.dataSource), 
+        SERVICES.getHeatmapTiles(siteKey, state.timespanType, zoom, state.categoryValue.name, state.datetimeSelection,
+                                bbox, Array.from(state.termFilters), undefined, Actions.DataSources(state.dataSource),
```

Comment:
Would it be possible to remove this undefined parameter? (see https://github.com/CatalystCode/project-fortis-interfaces/pull/10)